domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -738,6 +738,17 @@ public void save(String table, String id, Object entity, Handler<AsyncResult<Str
         /* returnId */ true, /* upsert */ false, /* convertEntity */ true, closeAndHandleResult(conn, replyHandler)));
   }
 
+  /**
+   * Insert entity into table.
+   * @param table database table (without schema)
+   * @param id primary key for the record
+   * @param entity a POJO (plain old java object)
+   * @param replyHandler returns any errors and the updated entity.
```

Comment:
Can you change "and the updated entity" to "and the entity after applying any database INSERT triggers"?